Pack-Smart/pack-smart-fe
# Feature/saved list verify deletion modal

## What's this PR do?
- Adds a verify deletion modal for when user clicks `x` on a saved list
- Makes `VerifyDeletion` modal dynamic

## Where should the reviewer start?
- `VerifyDeletion.js`, `SavedView.js`, `ListThumbnail.js`

## How should this be manually tested?
- From Saved List view, click to delete a list and make sure modal is popping up and is working correctly.

## Any background context you want to provide?
- no

## What are the relevant tickets?
- closes #99 

## Screenshots/GIFs
- 
<img width="749" alt="Screen Shot 2021-03-01 at 3 20 31 PM" src="https://user-images.githubusercontent.com/68157759/109553998-b0d4ee00-7aa1-11eb-820f-2f816169c44b.png">


## Questions:
- none

## Did you update the README?
- no
